client/app/src/main/java/com/example/mrides/Notification/InboxAdapter.java

```diff
@@ -25,11 +27,12 @@
  *
  */
 public class InboxAdapter extends RecyclerView.Adapter<InboxAdapter.ViewHolder> implements
-        View.OnClickListener{
+        View.OnClickListener, ActivityObserver{
 
     private Map<String,String> responseBody;
     private Context inboxContext;
     private Dialog dialog;
+    private RequestHandler requestHandler = new RequestHandler();
 
     public InboxAdapter(Context inboxContext){
```

Comment:
There should be a space between the closing parenthesis and opening curly bracket.